crypto/keyring/keyring.go

```diff
@@ -499,7 +499,7 @@ func (ks keystore) List() ([]*Record, error) {
 	var res []*Record //nolint:prealloc
 	sort.Strings(keys)
 	for _, key := range keys {
-		if strings.Contains(key, addressSuffix) {
+		if !strings.HasSuffix(key, infoSuffix) {
```

Comment:
This is a breaking change.

Reply:
I learned go from you, so I bet it is.

Thank you for your how to start guide.

I’ll try and clean this up is actually more likely I’ll speak with the author, I actually made this whole request kind of as a template to demonstrate how to make them, thank you very much for the review